api/src/main/java/com/cloud/event/EventTypes.java

```diff
@@ -87,6 +87,12 @@
 
 public class EventTypes {
 
+    public static final String EVENT_VMSCHEDULE_CREATE = "VM.SCHEDULE";
```

Comment:
Should this be called `VM.SCHEDULE.CREATE` ?

Reply:
Fixed

Reply:
I think this hasn't been fixed already. Also, can you please use the same prefix 'VM.SCHEDULE.' for all the related events?